sprout/appliances/models.py

```diff
@@ -1171,9 +1171,9 @@ def create(cls, owner, group, version=None, date=None, provider=None, num_applia
             time_leased=60, preconfigured=True, yum_update=False, container=False, ram=None,
             cpu=None, provider_type=None):
         if container:
-            container_q = ~Q(container=None) & ~Q(provider_type='openshift')
+            container_q = ~Q(container=None) & ~Q(provider__provider_type='openshift')
         else:
-            container_q = Q(container=None)
+            container_q = Q(container=None) | Q(provider__provider_type='openshift')
```

Comment:
Shouldn't these two branches be swapped? I am confused a bit.

Reply:
When we discussed this point last time, you mentioned that containers and openshift appliances are different stuff. So, it got me thinking that openshift appliances have to be attributed to regular appliances.

Reply:
@mfalesni please provide resolution to this comment

Reply:
@psav we were discussing this on IRC